toolkit/imageconfigs/linuxguard-amd64.yaml

```diff
@@ -119,14 +119,12 @@ os:
       - rd.luks=0
       - rd.hostonly=0
       - ipe.enforce=0
-      - systemd.journald.forward_to_console=1
 
   packages:
```

Comment:
Can we start a tracking spreadsheet that documents diff from steamboat line by line?

Reply:
Document started and shared with you